src/Clients/ProductsClient.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Creso\LaravelEasyofficeApi\Clients;
+
+class ProductsClient extends BaseClient
+{
+    public function get(string $uuid, bool $skipCache = false): array
+    {
+        return $this->getRequest("product/{$uuid}", skipCache: $skipCache);
```

Comment:
Named parameters are 🔥 but this one has not really an advantage, I would remove it.